ckb-bench/devtools/ci/script/benchmark.sh

```diff
@@ -2,22 +2,16 @@
 
 # ENVIRONMENT VARIABLES:
 # 
-#   * AWS_ACCESS_KEY, required, the AWS access key
-#   * AWS_SECRET_KEY, required, the AWS secret key
-#   * AWS_EC2_TYPE, optional, default is c5.xlarge, the AWS EC2 type
-#   * GITHUB_TOKEN, required, GitHub API authentication token
-
-set -euo pipefail
-
-AWS_ACCESS_KEY=${AWS_ACCESS_KEY}
-AWS_SECRET_KEY=${AWS_SECRET_KEY}
-AWS_EC2_TYPE=${AWS_EC2_TYPE:-"c5.xlarge"}
-GITHUB_TOKEN=${GITHUB_TOKEN}
-GITHUB_BRANCH=${GITHUB_BRANCH:-"develop"}
+GITHUB_BRANCH=${GITHUB_BRANCH:-"$GITHUB_REF_NAME"}
 GITHUB_REPO=${GITHUB_REPO:-"nervosnetwork/ckb"}
+BENCHMARK_ID=$GITHUB_RUN_ID
```

Comment:
GitHub Actions are not required for this script, it should be able to run locally.
You should pass `BENCHMARK_ID` to this script, but not `GITHUB_RUN_ID`.